mods/blackjack/web/style.css

```diff
@@ -124,7 +124,7 @@ body {
     width: 8vw;
```

Comment:
Would be nice to throw this fiel into prettier as well. And other files below as well.